pegaprox/core/manager.py

```
@@ -16648,6 +16674,18 @@ def _fetch_qemu_disk_usage(self, node: str, vmid: int) -> dict:
             return {}
 
+    def _fetch_qemu_memory(self, node: str, vmid: int):
+        """Get Linux guest memory pressure through QGA, or None if unavailable."""
+        if vmid in self._no_agent_vms:
+            return None
+        try:
+            from pegaprox.utils.guest_memory import get_guest_linux_memory
+            base = (f"https://{self.host}:{self.api_port}/api2/json/nodes/"
+                    f"{node}/qemu/{vmid}/agent")
+            return get_guest_linux_memory(self, base)
+        except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)